docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:bionic
+
+RUN apt update
+RUN apt -y install \
+		sudo \
+		# openssh-server \
+		git \
+		npm
+
+RUN useradd kaya --uid 1000 -m
+
+EXPOSE 4200
+
+WORKDIR /home/kaya
+USER kaya
+
+RUN git clone https://github.com/yoeriapts/kaya.git && \
```

Comment:
@yoeriapts can I ask why we are cloning the repository here when we can use `COPY`? Additionally, we also need a means of cloning and building the scilla binaries. You may wish to refer to https://github.com/Zilliqa/savant-ide/blob/545d132dab6cfe1e5931af5f29ed11847b5aac95/Dockerfile to get an idea of how this can be accomplished. Note that the reference `Dockerfile` is _not_ optimised at all, and can be significantly improved :)